data/desktop/Makefile

```diff
@@ -44,3 +51,6 @@ clean:
 	cat $< | \
 		sed s:@bindir@:$(BINDIR):g | \
 		cat > $@
+	cat $< | \
```

Comment:
Hm, this will not work, will it? The first cat will substitute the @bindir@ and the second will substitute @icondir@ but the second one does not take the first one as input but the original file? That seems to be why there is a test failure.

Reply:
Agh! Good catch! This is what happens when I just blindly copy/paste a piece of code O:-)

I'll fix it.

Reply:
Ok, now I think that it is fixed.